Utility.cpp

```diff
@@ -43,66 +43,68 @@ std::string getCharacterStats(Character* ch)
+        chPaladin->boostArmor (static_cast<int>( item.getBoost() * 1.3 ));
     }
-    else if( auto* ch = dynamic_cast<DragonSlayer*>(character))
+    else if( auto* chDSlayer = dynamic_cast<DragonSlayer*>(character))
     {
-        ch->boostArmor( item.getBoost() * 1.5 );
+        chDSlayer->boostArmor (static_cast<int>( item.getBoost() * 1.5 ));
     }
-    else if( auto* ch = dynamic_cast<Dragon*>(character) )
+    else if( auto* chDragon = dynamic_cast<Dragon*>(character) )
     {
         //dragons don't need defensive items
     }  
 }
-void useHelpfulItem(Character*, Item* item)
+void useHelpfulItem(Character* character, Item* item)
 {
-    if( auto* ch = dynamic_cast<Dwarf*>(character) )
+    if( auto* chDwarf = dynamic_cast<Dwarf*>(character) )
     {
-        ch->boostHitPoints(item->getBoost() * 2);
+        chDwarf->boostHitPoints(static_cast<int>(item->getBoost() * 2));
     }
-    else if( auto* ch = dynamic_cast<Paladin*>(character) )
+    else if( auto* chPaladin = dynamic_cast<Paladin*>(character) )
     {
-        ch->boostHitPoints(item->getBoost() * 1.5);
+        chPaladin->boostHitPoints(static_cast<int>(item->getBoost() * 1.5));
     }
-    else if( auto* ch = dynamic_cast<DragonSlayer*>(character))
+    else if( auto* chDSlayer = dynamic_cast<DragonSlayer*>(character))
     {
-        ch->boostHitPoints(item->getBoost() * 1.25);
+        chDSlayer->boostHitPoints(static_cast<int>(item->getBoost() * 1.25));
     }
-    else if( auto* ch = dynamic_cast<Dragon*>(character) )
+    else if( auto* chDragon = dynamic_cast<Dragon*>(character) )
     {
         //dragons don't carry helpful items!
     }
 }
-void useAttackItem(Character*, Item* item)
+void useAttackItem(Character* character, Item* item)
 {
-    if( auto* ch = dynamic_cast<Dwarf*>(character) )
+    if( auto* chDwarf = dynamic_cast<Dwarf*>(character) )
     {
-        ch->boostAttackDamage(item->getBoost() * 1.5);
+        chDwarf->boostAttackDamage(static_cast<int>(item->getBoost() * 1.5));
     }
-    else if( auto* ch = dynamic_cast<Paladin*>(character) )
+    else if( auto* chPaladin = dynamic_cast<Paladin*>(character) )
     {
-        ch->boostAttackDamage(item->getBoost() * 1.33);
+        chPaladin->boostAttackDamage(static_cast<int>(item->getBoost() * 1.33));
     }
-    else if( auto* ch = dynamic_cast<DragonSlayer*>(character))
+    else if( auto* chDSlayer = dynamic_cast<DragonSlayer*>(character))
     {
-        assert(false);
+        //assert(false);
         //DragonSlayers get a 10x boost when attacking dragons, from their attack item.
         //so their attack item should boost their attack damage by a factor of 10
         //this means you need to GET the attack damage, multiply it by the item's boost, and BOOST the attackDamage with that multiplied value.  
         //check Character.h for available member functions you can use.
+		chDSlayer->boostAttackDamage(chDSlayer->getAttackDamage()*item->getBoost());
+
```

Comment:
Fix this indentation
add space around the `*` to improve code readability
- [ ] revise